go/api/commands.go

```diff
@@ -442,6 +442,25 @@ type StringCommands interface {
 	//
 	//[valkey.io]: https://valkey.io/commands/getdel/
 	GetDel(key string) (Result[string], error)
+	// Del removes the specified keys from the database. A key is ignored if it does not exist.
+	//
+	// @apiNote When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
```

Comment:
What is the golang doc syntax for such things?
```suggestion
	// Note:
	//  When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
```